src/roles/foreman_proxy/tasks/main.yaml

```diff
@@ -0,0 +1,66 @@
+---
+- name: Create Foreman Proxy directories
+  ansible.builtin.file:
+    path: "{{ item }}"
+    state: directory
+    owner: root
+    group: root
+    mode: '0755'
+  loop:
+    - /var/log/foreman-proxy
```

Comment:
We shouldn't create `/var/log/foreman-proxy` and instead configure it to log to stdout or journald